.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,57 +1,131 @@
+
+Add only ONE sig here via a /sig-primary <name> entry.
 -->
+/sig-primary ???
+
+> DISCUSSION REQUIRED
+>
+> Should we use the above pattern and update the label prow plugin ? https://github.com/kubernetes/test-infra/tree/master/prow/plugins/label. Logic would have to ensure that there is only a single primary sig associated. This metadata can be leveraged by other processes outside of the release notes.
+> 
+> or, if this is ONLY relevant to release notes then, we can add here as a `` ```primary-sig`` code block.
 
 **What type of PR is this?**
-> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
+<!-- 
+Label this pull request according to what type of issue you are addressing, especially if this is a release targeted pull request. 
+
+For reference on required PR labels, you can find more details at:
+https://git.k8s.io/community/contributors/devel/sig-release/release.md#issuepr-kind-label
+
+Add only ONE of the following /kind <type> entries.
+
+/kind api-change
+/kind bug
+/kind cleanup
+/kind design
+/kind documentation
+/kind failing-test
+/kind feature
+/kind flake
+-->
+/kind ???
+
+> DISCUSSION REQUIRED
 >
-> /kind api-change
-> /kind bug
-> /kind cleanup
-> /kind design
-> /kind documentation
-> /kind failing-test
-> /kind feature
-> /kind flake
+> Should we include a kind type of `deprecation`? And should we then include a `` ```depracation-notes`` code block to provide specific detailed notes around the depracation.
+> 
+> Should we only allow a single kind to be specified? There are a number of examples where multiple kind values make sense.
 
 **What this PR does / why we need it**:
+<!--
+Provide a description on what this PR does and/or why it is required.
+-->
+Description here ...
 
 **Which issue(s) this PR fixes**:
 <!--
-*Automatically closes linked issue when PR is merged.
-Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
-_If PR is about `failing-tests or flakes`, please post the related issues/tests in a comment and do not use `Fixes`_*
+Add references to issues that are fixed by this PR by using one or more of the following templates:
+
+Fixes #<issue number>
+Fixes <issue url>
+
+NOTE: The linked issue will automatically be closed when the PR is merged.
+
+If the PR is about failing-tests or flakes, please post the related issues/tests in a comment and do not use the Fixes templates above.
+-->
+Fixes #???
+
+**Have you added or run tests for your PR?**
+<!--
+Indicate whether you have added or run the appropriate tests for your PR by using ONE of the following responses:
+
+N/A
+Yes
+No
+
+More details can be found in the Testing Guide:
+https://git.k8s.io/community/contributors/devel/sig-testing/testing.md
 -->
-Fixes #
+No
 
 **Special notes for your reviewer**:
+<!--
+Provide any additional notes that may be helpful to the PR reviewers.
+-->
+None
```

Comment:
Would prefer to leave this as-is because adding a "special note" would first mean removing the `None`. Overall, just want to reduce the amount of work that the author would need to do :)

Reply:
I thought it would make it clearer when reading the PR - so seeing `None` instead of a blank line, but I understand the need to reduce any additional overhead for the PR submitter. I'll remove this.